controllers/cache_handler_test.go

```diff
@@ -139,7 +139,7 @@ var _ = Describe("Cache Handler verifications", func() {
 
 			By("ensuring created event was fired")
 			event := findEvent(disruption.CreationTimestamp.Time, v1beta1.Events[v1beta1.EventDisrupted], eventList.Items, targetPodA.Name)
-			Expect(event).ToNot(BeNil())
+			Eventually(event).ShouldNot(BeNil())
```

Comment:
dyk eventually can take values, and not just functions? I didn't until today

Reply:
Didn't knew either 💪 
I'm not sure I get how this change could help reducing the flakiness however 🤔 
We will still call `findEvent` once, and either the returned value will be valid or not, leading to the same as before?

Looking at the doc:
* https://onsi.github.io/gomega/#category-1-making-codeeventuallycode-assertions-on-values

Using Eventually with a func, by calling findEvent several time could fix the flakiness if it was caused by a too early call however.

Not sure I get everything, if I missed something please let me know 🤗

Reply:
TIL 💪 
Shouldn't we pass `findEvent` as a func to `Eventually`?

Haha, just realize Matthieu said the same thing.